spec/classes/hdm_docker_spec.rb

```diff
@@ -12,19 +22,34 @@
           'version' => '1.0.1',
         }
       end
-      let(:pre_condition) do
-        if facts[:os]['family'] == 'Suse'
-          '
-            class { "docker":
-              acknowledge_unsupported_os => true,
-            }
-          '
-        end
+
+      it { is_expected.to compile.with_all_deps }
+      it { is_expected.to contain_class('hdm::docker') }
+      it { is_expected.to contain_class('docker') }
+      it { is_expected.to contain_docker__image('ghcr.io/betadots/hdm') }
```

Comment:
maybe add here a `.with('version' => 'x.y.z')` to ensure the version from the variable in the class is parsed?

Reply:
Done